examples/computer_vision/_mnist_pl/model_def.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
This example will not be part of the PR that lands, right?  I'm guessing that's why `_mnist_py` starts with a `_`

Reply:
it is why? depending on when we land it it might initially be with the _

Reply:
@rb-determined-ai The example will land as part of the PR. And let's remove the underscore in the file names.